src/app/globals.css

```
@@ -1,126 +1,227 @@
-  --sidebar-foreground: oklch(0.985 0 0);
-  --sidebar-primary: oklch(0.488 0.243 264.376);
-  --sidebar-primary-foreground: oklch(0.985 0 0);
-  --sidebar-accent: oklch(0.269 0 0);
-  --sidebar-accent-foreground: oklch(0.985 0 0);
-  --sidebar-border: oklch(1 0 0 / 10%);
-  --sidebar-ring: oklch(0.556 0 0);
+  /* Background */
+  --background: var(--neutral-900);
+  --foreground: var(--neutral-0);
+
+  /* Primary*/
```

Stylelint (17.2.0)

[error] Expected whitespace before "*/" (comment-whitespace-inside)

(comment-whitespace-inside)